lib/users.ts

```diff
@@ -116,6 +167,7 @@ export const getUsers = async (ability: UserAbility) => {
           id: true,
           name: true,
           email: true,
+          active: true,
```

Comment:
I know it's somewhat out of scope but since we're updating the function we could maybe also update the fn return type?

Reply:
Perhaps there's a better way to define the SelectedUser type but added now.